### PR TITLE
Added short section on zdb to Command Line Utilities under freenas-debug with correct cache location flag

### PR DESCRIPTION
**Pull request by ZackaryWelch**

### CODE REVIEW

File: userguide/cli.rst [hunk 1]
```diff
@@ -985,6 +985,17 @@ To collect the output of every module, use :literal:`-A`:
 
    freenas-debug -A
 
+For collecting debug information about a single pool, use
+:command:`zdb` with :literal:`-U /data/zfs/zpool.cache`
+followed by the name of the pool:
```
Comment (wblock) [inline userguide/cli.rst]: In the old UI, we are still using the term "volume".  The change to this branch should mention both:
"followed by the name of the volume (ZFS pool):"

File: userguide/cli.rst [hunk 2]
```diff
@@ -985,6 +985,17 @@ To collect the output of every module, use :literal:`-A`:
...
+
+.. code-block:: none
+
+    zdb -U /data/zfs/zpool.cache pool1
```
Comment (wblock) [inline userguide/cli.rst]: In master, I think the example volume name is "volume1". Please check.

Comment (ZackaryWelch) [reply userguide/cli.rst]: Actually pool1 is used much more often than volume1 apparently

Comment (ZackaryWelch) [reply userguide/cli.rst]: Changed it to volume1 anyway.

**Pull request merged**